### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by tynorton**
- RD-2158 - Updated French Localization
- Setting Processing Strings

### CODE REVIEW

File: src/DDTech.Localization/Common.es.resx [hunk 1]
```diff
@@ -136,7 +136,7 @@
     <value>Borrar</value>
   </data>
   <data name="Button_OK" xml:space="preserve">
-    <value>OK</value>
+    <value>Vale</value>
```
Comment (efalsken) [inline src/DDTech.Localization/Common.es.resx]: mistake?

File: src/DDTech.Localization/Common.fi.resx [hunk 2]
```diff
@@ -208,7 +211,7 @@
     <value>Matala</value>
   </data>
   <data name="Priority_Medium" xml:space="preserve">
-    <value>Medium</value>
+    <value>Normaali</value>
```
Comment (efalsken) [inline src/DDTech.Localization/Common.fi.resx]: Mistake?

File: src/DDTech.Localization/Common.ko.resx [hunk 3]
```diff
@@ -136,7 +136,7 @@
     <value>삭제</value>
   </data>
   <data name="Button_OK" xml:space="preserve">
-    <value>OK</value>
+    <value>그래</value>
```
Comment (efalsken) [inline src/DDTech.Localization/Common.ko.resx]: Mistake?

File: src/DDTech.Localization/Common.es.resx (hunk 1 above)
Comment (tynorton) [inline src/DDTech.Localization/Common.es.resx]: Yes, good catch.

File: src/DDTech.Localization/Common.fi.resx (hunk 2 above)
Comment (tynorton) [inline src/DDTech.Localization/Common.fi.resx]: I may let this one through, as the other priorities are not locked `en`

**Pull request merged**